airbyte-commons/src/main/resources/log4j2.xml

```diff
@@ -2,40 +2,69 @@
 <Configuration status="INFO">
     <Properties>
 
-        <!-- The following patterns mask the string apikey=<string> to apikey=***** to prevent secrets leaking. -->
-        <Property name="default-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %highlight{%p} %C{1.}(%M):%L - %X - %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
-        <!-- The following patter log the application name in the beginning of the line, then a regular log line after a - separator. If the application name is not present in the context, the separator is removed -->
-        <Property name="docker-worker-file-pattern">%replace{%X{log_source} - }{^ - }{}%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %p (%X{job_root}) %C{1}(%M):%L - %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
-        <!-- Remove paths from logs on Cloud as users aren't able to access file paths and these end up being noise. -->
-        <Property name="cloud-worker-file-pattern">%replace{%X{log_source} - }{^ - }{}%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %p %C{1}(%M):%L - %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
+        <!-- Mask the string apikey=<string> to apikey=***** to prevent secrets leaking. -->
+        <Property name="default-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %highlight{%p} %C{1.}(%M):%L - %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
+        <!--Logs the timestamp and log_source/application name in the beginning of the line if it exists with a > separator, and then always the rest of the line.-->
+        <Property name="simple-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0}%replace{ %X{log_source}}{^ -}{} > %replace{%m}{apikey=[\w\-]*}{apikey=*****}%n</Property>
 
         <!-- Always log INFO by default. -->
-        <Property name="log-level">$${env:LOG_LEVEL:-INFO}</Property>
+        <Property name="log-level">$${sys:LOG_LEVEL:-${env:LOG_LEVEL:-INFO}}</Property>
 
-        <Property name="s3-bucket">$${env:S3_LOG_BUCKET}</Property>
-        <Property name="s3-region">$${env:S3_LOG_BUCKET_REGION}</Property>
-        <Property name="s3-aws-key">$${env:AWS_ACCESS_KEY_ID}</Property>
-        <Property name="s3-aws-secret">$${env:AWS_SECRET_ACCESS_KEY}</Property>
-        <Property name="s3-minio-endpoint">$${env:S3_MINIO_ENDPOINT}</Property>
-        <Property name="s3-path-style-access">$${env:S3_PATH_STYLE_ACCESS}</Property>
+        <Property name="s3-bucket">$${sys:S3_LOG_BUCKET:-${env:S3_LOG_BUCKET}}</Property>
```

Comment:
to make sure I understand, this looks at the env var first then the sys var?

Reply:
No, this looks at sys first and then env var.

Reply:
I thought it made sense in that direction so we could always override runtime env vars with something we set in a property.

Reply:
gotcha.

makes sense. can you leave a comment here? XML is hard to parse otherwise.